app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_save :update_post_likes_counter
+  def update_post_likes_counter
+    posts.increment!(:likes_counter)
+  end
```

Comment:
- [ ] Well done using after_save  to call the #update_post_likes_counter  method every time a new post has been saved to database, I just recommend to make the update_post_likes_counter method private.